python/lsst/dax/imgserv/fitsMetadataSchema.py

```diff
@@ -0,0 +1,59 @@
+
+
```

Comment:
I just renamed this file and stuck the schema into a list. It wasn't there before :). I'll add it.